pom.xml

```diff
@@ -577,7 +577,7 @@
       <dependency>
         <groupId>${project.groupId}</groupId>
         <artifactId>netty-tcnative</artifactId>
-        <version>1.1.32.Fork1</version>
+        <version>1.1.32.Fork2-SNAPSHOT</version>
```

Comment:
Mostly so that I can pick up the SSL_OP_NO_TLSv1_1 change, that but is preventing me from porting to netty-tcnative for some internal projects :)